### PR TITLE
Support generation of Unoptimised VMs

### PR DESCRIPTION
**Pull request by PalumboN**
Work done while trying to generate different versions of the VM.

- Adding script methods for model generation
- Adding missing definitions in some visitors (arrived by unoptimised code)
- Adding non-supported cases for PhiSimplification (TODO: open issue)
- Support staged compilation of more instructions :)

### CODE REVIEW

File: Druid/DRBytecodeIRGenerator.class.st [hunk 1]
```diff
@@ -232,7 +232,7 @@ DRBytecodeIRGenerator >> interpretInstructionPointerWith: aRBVariableNode [
 DRBytecodeIRGenerator >> interpretInterruptCheckWith: aRBMessageNode [
 
 	^ (self
-		   addInstructionWithNoResultFrom: aRBMessageNode
+		   addInstructionFrom: aRBMessageNode
```
Comment (guillep) [inline Druid/DRBytecodeIRGenerator.class.st:235]: Here we are losing the "NoResult" is that ok?

File: Druid/DRInterpreterToCompiler.class.st [hunk 2]
```diff
@@ -121,11 +127,25 @@ DRInterpreterToCompiler class >> generateDruidJITModel [
 		doFailOnFirst;
 		selectPrimitives: [ :e | primitives includes: e selector ];
 		selectBytecodes: [ :e | bytecodes includes: e bytecodeNumberStart ];
-		targetClass: DruidJIT;
+		targetClass: targetClass;
```
Comment (guillep) [inline Druid/DRInterpreterToCompiler.class.st:130]: :)

File: Druid/DRInterpreterToCompiler.class.st [hunk 3]
```diff
@@ -121,11 +127,25 @@ DRInterpreterToCompiler class >> generateDruidJITModel [
...
+
+	<script>
+	self generateDruidJITModelOn: DruidCogObjectRepresentationFor64BitSpur "Compiles in the ObjectRepresentation hierarchy"
+]
+
+{ #category : #'instance creation' }
+DRInterpreterToCompiler class >> generateUnoptimisedDruidJITModel [
+
+	<script>
+	self generateDruidJITModelOn: DruidUnoptimisedJIT
```
Comment (guillep) [inline Druid/DRInterpreterToCompiler.class.st:146]: :) :)

**Review by guillep — APPROVED** [review]
Ok, let's go and move forward!

File: Druid/DRBytecodeIRGenerator.class.st (hunk 1 above)
Comment (PalumboN) [reply Druid/DRBytecodeIRGenerator.class.st:235]: Yes, it has result

**Pull request merged**